lib/cmxl/fields/vmk_summary.rb

```diff
@@ -0,0 +1,37 @@
+module Cmxl
+  module Fields
+    class VmkSummary < Field
+      self.tag = 90
+      self.parser = /(?<entries>\d{,53})(?<currency>\w{3})(?<amount>[\d|,|\.]{1,15})/i
+
+      def credit?
+        modifier == 'C'
+      end
+
+      def debit?
+        modifier == 'D'
+      end
+
+      def entries
+        data['entries'].to_i
+      end
+
+      def currency
+        data['currency']
+      end
+
```

Comment:
see notes in `FloorLimitIndicator`

`#currency` should already be handled by: https://github.com/railslove/cmxl/blob/master/lib/cmxl/field.rb#L120

and what's the storey with the `.to_s.casecmp('C').zero?`